src/aura/TasksTableComponent/TasksTableComponent.cmp

```diff
@@ -0,0 +1,42 @@
+<aura:component controller="TaskItemController" implements= "force:appHostable, flexipage:availableForAllPageTypes" >
+  <aura:handler name="init" value="{!this}" action="{!c.doInit}"  />
+  <aura:handler name="reRenderTaskTable" event="c:UpdateTaskStatus" action="{!c.doInit}" />
```

Comment:
reloading all tasks just to remove one completed task from the list is very sub-optimal as a solution. Your event should contain the completed record as a parameter and the action handling it should simply locate the task in the right list and remove it (all done in the front-end).

Reply:
Ready